### ISSUE TITLE
On sign up page, display email that we've sent a message too

### ISSUE THREAD

**Issue by JosephMcArthur** [OAButton/discussion#343]
- I just sign up with an error in my email & I couldn't tell, just got confused as to why the email took so long. 
- I also got distracted while going to my inbox & forgot what email I'd sent it too so didn't know where to check. 

Both of these two things seem like things that could happen to users. Both would be solved by displaying the email we've just the message too on the screen. Could be like:

> If you don't receive an email to <user's email> within a few minutes, check your spam folder. Contact us at help@openaccessbutton.org if you don't receive an email. 

Not suggesting we do it, just leaving a note here for discussion. Seems like a win-win though.

**Comment by JosephMcArthur** [comment]
Could do this with #424

**Comment by JosephMcArthur** [comment]
@chealsye can you advise on what copy you might want to see changed on our current page to see this implemented?

**Comment by JosephMcArthur** [comment]
An implementation of this is currently on the dev site

**Comment by chealsye** [comment]
@JosephMcArthur I didn't miss anything with this, did I? When I first saw 23 October my stomach dropped. The labels and assignments prior to 6 hours ago look like I wouldn't have been alerted, but I wanted to check - don't want the adjustments I made to github alerts result in things falling through the cracks.

What your suggestion is just adding the email address, right? Our page already includes the rest. I'm fine with it - but @markmacgillivray, can it work? Can we display the users email right after they submit on the "Go check your email!" page?
